src/System.ComponentModel.TypeConverter/tests/TypeDescriptorTests.cs

```diff
@@ -66,6 +67,22 @@ public void GetAssociationReturnsExpectedObject()
             Assert.Equal(secondaryObject, associatedObject);
         }
 
+        [Fact]
+        public void GetAssociationReturnsDesigner()
+        {
+            var designer = new MockDesigner();
+            var designerHost = new MockDesignerHost();
+            var component = new DescriptorTestComponent();
+
+            designerHost.AddDesigner(component, designer);
+            component.AddService(typeof(IDesignerHost), designerHost);
+
+            var associatedObject = TypeDescriptor.GetAssociation(designer.GetType(), component);
+
+            Assert.IsType(designer.GetType(), associatedObject);
+            Assert.Equal(designer, associatedObject);
```

Comment:
Nit: I'm not sure what the semantics of `GetAssociation` are, but could this be `Assert.Same` instead of `Assert.Equal`?

Reply:
Yup